### PR TITLE
Set up CI with Azure Pipelines

### PR DESCRIPTION
**Pull request by pkanher617**

### CODE REVIEW

File: lib/msal-core/azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Node.js
+
+# Build a general Node.js project with npm.
+# Add steps that analyze code, save build artifacts, deploy, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
+
+jobs:
+  - job: rolling_VS2017_build
+    displayName: 'Build Job for JS (VS2017)'
```
Comment (DarylThayil) [inline lib/msal-core/azure-pipelines.yml]: what is vs 2017

**Comment by DarylThayil** [comment]
is publish in this or only build?

File: lib/msal-core/azure-pipelines.yml (hunk 1 above)
Comment (jasonnutter) [reply lib/msal-core/azure-pipelines.yml]: Visual Studio 2017

Comment (pkanher617) [reply lib/msal-core/azure-pipelines.yml]: This is the server that we are using to build.

**Comment by pkanher617** [comment]
Publish is in VSTS as a separate task.

File: lib/msal-core/build/npm-build-steps.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+steps:
+- task: NodeTool@0
+  inputs:
+    versionSpec: '10.x'
+  displayName: 'Install Node.js'
+
+- task: Npm@1
+  inputs:
+    command: 'install'
```
Comment (jasonnutter) [inline lib/msal-core/build/npm-build-steps.yml]: If you have it run `npm ci` instead of `npm install` it'll go faster.

**Pull request merged**